br/pkg/version/version.go

```diff
@@ -165,6 +165,14 @@ func CheckVersionForBRPiTR(s *metapb.Store, tikvVersion *semver.Version) error {
 				s.Address, tikvVersion, build.ReleaseVersion)
 		}
 	}
+
+	if BRVersion.Major > 8 || (BRVersion.Major == 8 && BRVersion.Minor >= 4) {
+		if tikvVersion.Major < 8 || (tikvVersion.Major == 8 && tikvVersion.Minor < 4) {
+			return errors.Annotatef(berrors.ErrVersionMismatch,
+				"TiKV node %s version %s is too old because the PITR id map is written into the cluster system table mysql.tidb_pitr_id_map, please use the tikv with version v8.4.0+",
```

Comment:
I guess TiKV isn't related to the systable...? Perhaps replace "TiKV" with "cluster".

Reply:
Yes, there seems an assumption that each component version is the same.